### PR TITLE
sosivio rework & unpin 1.5.3

### PR DESCRIPTION
**Pull request by DanArlowski**
### Thank you for making MicroK8s better

Please reference the issue this PR is fixing, or provide a description of the problem addressed.

*Also verify you have:*
* [x] Read the [contributions](https://github.com/ubuntu/microk8s/blob/master/CONTRIBUTING.md) page.
* [x] Submitted the [CLA form](https://ubuntu.com/legal/contributors/agreement), if you are a first time contributor.

### CODE REVIEW

**Comment by DanArlowski** [comment]
hey @ktsakalozos 
I have reworked the scripts after we released

Thanks!


because we released the new version, this also address https://github.com/canonical/microk8s-community-addons/issues/156

File: addons/sosivio/enable [hunk 1]
```diff
@@ -2,58 +2,192 @@
...
-do
-  sleep 1
+  echo ""
+  echo -e "${BLUE}Enabling Sosivio${NC}"
+  echo ""
+
+  print_debug "  creating $NAMESPACE_SOSIVIO namespace"
+  $KUBECTL create ns $NAMESPACE_SOSIVIO > /dev/null 2>&1 || true
+
+  HELM_CMD="$HELM upgrade --install -n $NAMESPACE_SOSIVIO \
```
Comment (ktsakalozos) [inline addons/sosivio/enable:93]: Could we please make sure we install a specific version of sosivio so we do not have this problem in the future? This version should also be set in the top level addons.yaml file describing the addon.

Comment (ktsakalozos) [reply addons/sosivio/enable:93]: Any update on this @DanArlowski ?

**Comment by DanArlowski** [comment]
hey @ktsakalozos ,
sorry for the delay.
after pinning a version - how will it be possible to update versions?
as i recall a specific commit is attached to specific microk8s release.

**Comment by ktsakalozos** [comment]
> after pinning a version - how will it be possible to update versions?
> as i recall a specific commit is attached to specific microk8s release.

That's right. In this repository there are branches for each release (eg, 1.26, 1.27). To update the sosivio version on a specific MicroK8s track you need to put up a PR against the respective branch.

**Comment by DanArlowski** [comment]
@ktsakalozos 
LGTM
pinned to 1.7.1

**Comment by ktsakalozos** [comment]
Thank you for this work @DanArlowski . We will review this PR soon and it will go live with the 1.29 release. 

We will also review your PR against the 1.27 branch. Maybe you want to patch the 1.28 branch too. Many thanks.

**Comment by DanArlowski** [comment]
thanks,

Is there an easier way to merge new releases rather than doing so for each seperate microk8s branch?

that was why i was kind of against pinning :)

File: addons/sosivio/enable [hunk 2]
```diff
@@ -2,58 +2,191 @@
...
+if [ $DEBUG -eq 1 ]; then 
+  echo "debug flag set" >&2
+  echo "repo url is set to $HELM_REPO_URL"
+  echo "repo name is set to $HELM_REPO_NAME"
+fi
+
+install_addons_prereq
+install_soivio
+wait_for_sosivio
+print_finished
```
Comment (berkayoz) [inline addons/sosivio/enable:191]: I think we forgot to call the `add_helm_repo` function.

```suggestion
install_addons_prereq
add_helm_repo
install_soivio
wait_for_sosivio
print_finished
```

**Review by berkayoz — CHANGES_REQUESTED** [review]
Hey @DanArlowski, thank you for your efforts. Everything seems good for the most part, one small comment.

File: addons/sosivio/enable (hunk 2 above)
Comment (berkayoz) [reply addons/sosivio/enable:191]: We should probably update respective version PRs #175 and #176 with this change as well.

Comment (ktsakalozos) [reply addons/sosivio/enable:191]: Any update on this @DanArlowski ?

**Review by ktsakalozos — APPROVED** [review]
A few minor fixes I see, I will address in a follow up PR.

**Pull request merged**